src/utils/azureUtils.ts

```diff
@@ -72,7 +72,7 @@ export async function pollAzureAsyncOperation(restResponse: msRest.RestResponse,
     request.prepare({ method: 'GET', url });
     await credentials.signRequest(request);
 
-    const client: msRest.ServiceClient = createGenericClient();
+    const client: msRest.ServiceClient = await createGenericClient();
```

Comment:
Was this always a promise or did that get changed in the ui package?

Reply:
It was changed in this release: https://github.com/microsoft/vscode-azuretools/releases/tag/v0.36.0-ui